src/main/java/com/movierecommender/project/dto/Signin.java

```diff
@@ -0,0 +1,42 @@
+package com.movierecommender.project.dto;
+
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public class Signin {
+
+    @NotBlank(message = "Username must not be blank")
+    @Size(min = 3, max = 20, message = "Username must be between 3 and 20 characters")
+    private String username;
+
+    @NotBlank(message = "Password must not be blank")
+    @Size(min = 6, message = "Password must be at least 6 characters")
```

Comment:
added to create error messages for the user when inputting information to sign in